.github/dependabot.yml

```diff
@@ -27,3 +27,17 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+    ignore:
+    # Ignore gcp dependencies
+    - dependency-name: "com.google.api:gax*"
+    - dependency-name: "com.google.cloud.bigdataoss:*"
+    - dependency-name: "com.google.cloud:google-cloud-spanner*"
+    - dependency-name: "com.google.code.gson:*"
+    - dependency-name: "com.google.oauth-client:*"
```

Comment:
Can you also add `com.google.http-client:*`?

Reply:
Done!